src/VirtoCommerce.DemoSolutionFeaturesModule.Data/Caching/Customer/DemoTaggedMemberCacheRegion.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Primitives;
+using VirtoCommerce.DemoSolutionFeaturesModule.Core.Models.Customer;
+using VirtoCommerce.Platform.Core.Caching;
+
+namespace VirtoCommerce.DemoSolutionFeaturesModule.Data.Caching.Customer
+{
+    public class DemoTaggedMemberCacheRegion : CancellableCacheRegion<DemoTaggedMemberCacheRegion>
```

Comment:
Try to make static

Reply:
The static class should be derived from the object.